Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -1899,6 +1900,10 @@ -(void)togglePinStatusAction:(id)sender
 }
 
 - (void)viewVersionAction:(id)sender {
+    if (bSearching) {
+        [self endSearching:_noteEditorTextView];
+    }
```

Comment:
Annoying comment: worth moving to a method? dunno, `ensureSearchIsDismissed` or something like that?

Reply:
Good point! Indeed, it will make it more clear 👍